_data/README.md

```diff
@@ -0,0 +1,14 @@
+# Projektdaten
+
+Projektdaten werden nach diesem Schema erfasst:
+
+```
+'search_it': {'lead': 'tyrant88', 'transferred_at': '2017-02-05'}
+```
+
+* {Key}  
+Entspricht dem Namen des Repos.
+* __`lead`__  
```

Comment:
sollten wir diese doku ggf. direkt in _data/projects.yml als kommentar hinterlegen?